packages/zero-height/package.json

```diff
@@ -45,6 +46,7 @@
   },
   "devDependencies": {
     "@types/node": "^16.9.0",
+    "tslib": "^2.3.1",
```

Comment:
Pref: Put common dependencies in the root package.json

Reply:
I tried that, but hoisting is not supported when using yarn workspaces 😅 

```lerna ERR! EWORKSPACES --hoist is not supported with --npm-client=yarn, use yarn workspaces instead```

Reply:
Fek, I'll look into somewhere in the future